yarn.lock

```diff
@@ -762,6 +762,93 @@
     ajv "^8.0.0"
     ajv-formats "^2.0.0"
 
+"@jupiterone/integration-sdk-cli@^8.31.1":
```

Comment:
Release of 9.0.0 failed which caused yarn to reinstall 8.31.1... Will fix once 9.0.0 is released.